ci-operator/config/openshift/os/openshift-os-master.yaml

```diff
@@ -125,12 +125,12 @@ promotion:
 releases:
   initial:
     integration:
-      name: "4.11"
+      name: "4.12"
```

Comment:
This doesn't seem to match some of the other CI configs (MCO, Installer) for the `master` branch that I checked and I'm curious if this is going to have an adverse affect on the automated branching operation.  Any harm in leaving it at 4.11?